.github/workflows/workflow-publish-nuget.yml

```diff
@@ -0,0 +1,56 @@
+name: "Publish nuget package"
+
+on:
+  workflow_call:
+    inputs:
+      version:
+        description: "Version"
+        required: true
+        type: string
+    secrets:
+      NUGET_API_TEST_KEY:
+        required: true
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    timeout-minutes: 15
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: set PROJECT variable to point to project
+        run: |
+          PROJECT=$(find . -name '*Digdir.Library.Dialogporten.WebApiClient.csproj' -printf "%p"  -quit)
```

Comment:
Fin og generisk workflow dette! Det gjør at vi kan gjør som når vi publiserer npm-pakker, å gjøre denne enda mer generisk ved å ta "path" som input-parameter.

Reply:
Lagg også til en source som har nugettest som default for å gjøre enda mer generisk.